sync.py

```diff
@@ -4,6 +4,7 @@
 
 import clustering
 import storage
+import stability_score as ss
```

Comment:
Why do you need this here?

Reply:
Oh I forgot to remove this. Thanks for pointing that out.